src/hydro_system.hpp

```diff
@@ -1026,6 +1028,8 @@ void HydroSystem<problem_t>::ComputeFluxes(amrex::MultiFab &x1Flux_mf, amrex::Mu
 		sL.cs = cs_L;
 		sL.E = E_L;
 		sL.Eint = Eint_L;
+		sL.by = 0.0;
+		sL.bz = 0.0;
```

Comment:
Can you add a comment here that explains that you've just set the magnetic fields to zero for testing purposes?